### DISCUSSION TITLE
Exchange OWA and Authelia

### DISCUSSION THREAD

**Discussion by Tartiflette-glpi** [authelia/authelia#6684] [Q&A] [answered]
Hi,
I'm wondering how to access an Exchange OWA (on premise) with Authelia?
Today, I have an access to my OWA with an Nginx Proxy Manager.
I've installed and configured an Authelia and I would like to secure the web access of my OWA.
The problem when i'm trying it is that I can't see or reply to mails when Authelia is active with my OWA.
When I delete the Proxy host's advanced configuration (authelia / MFA), it works again...

Do I need to configure anything specific in my Proxy Host?
80, 81 and 443 Ports are published in my portainer.

**Comment by james-d-elliott** [comment, marked as answer]
There may be requests sent to OWA that don't include the cookie. If so you'd have to add a bypass rule for those requests. They should be logged. Unfortunately there is not a whole lot we can do to fix that otherwise.

**Comment by Smiley-k** [comment]
> Hi, I'm wondering how to access an Exchange OWA (on premise) with Authelia? Today, I have an access to my OWA with an Nginx Proxy Manager. I've installed and configured an Authelia and I would like to secure the web access of my OWA. The problem when i'm trying it is that I can't see or reply to mails when Authelia is active with my OWA. When I delete the Proxy host's advanced configuration (authelia / MFA), it works again...
> 
> Do I need to configure anything specific in my Proxy Host? 80, 81 and 443 Ports are published in my portainer.


Can you tell me how you did it? Would you like to share your advice on how you did it?

**Reply by jersam** [reply]
https://blog.cortanadesign.com.au/2025/09/10/how_to_get_exchange_2019_working_behind_nginx_proxy_manager

might help you.

**Comment by jersam** [comment]
> Hi, I'm wondering how to access an Exchange OWA (on premise) with Authelia? Today, I have an access to my OWA with an Nginx Proxy Manager. I've installed and configured an Authelia and I would like to secure the web access of my OWA. The problem when i'm trying it is that I can't see or reply to mails when Authelia is active with my OWA. When I delete the Proxy host's advanced configuration (authelia / MFA), it works again...
> 
> Do I need to configure anything specific in my Proxy Host? 80, 81 and 443 Ports are published in my portainer.

Hey. Semi-unrelated question how do you use OWA with NPM? do you break out ActiveSync for Outlook clients? Only force webmail?
Funny as it is... i googled today found this https://blog.cortanadesign.com.au/2025/09/10/how_to_get_exchange_2019_working_behind_nginx_proxy_manager lovely. =R

I use Duo on my OWA/ECP requests for on-prem exchange.
